ccsds/TC-Packet.acn

```diff
@@ -38,23 +38,40 @@ TC-PacketPrimaryHeader []
     packetDataLength []
 }
 
-TC-PacketDataField []
+TC-PacketDataField <INTEGER:secondaryHeaderFlag> []
 {
-    packetSecondaryHeader [],
-    userData [align-to-next byte]
+--! packetSecondaryHeader [present-when secondaryHeaderFlag],
+    dataWithSecondaryHeader [present-when secondaryHeaderFlag==1]
+    {
+        secondaryHeader [],
+        userData <secondaryHeader.messageType-ID.serviceType-ID,
+                  secondaryHeader.messageType-ID.messageSubtype-ID,
+                  secondaryHeaderFlag> []
```

Comment:
so now this userData should not used secondaryHeaderFlag, because it always be 1